src/components/InvertedFlatList/BaseInvertedFlatList/index.e2e.tsx

```diff
@@ -1,6 +1,6 @@
 import React, {forwardRef, useMemo} from 'react';
 import type {FlatListProps, ScrollViewProps, ViewToken} from 'react-native';
-import {FlatList} from 'react-native';
+import {DeviceEventEmitter, FlatList} from 'react-native';
```

Comment:
DeviceEventEmitter is [deprecated](https://github.com/facebook/react-native/blob/c39538b79dbc7f84b536431ac2230d259f846e42/packages/react-native/Libraries/EventEmitter/RCTNativeAppEventEmitter.js#L14), but we can use NativeEventEmitter instead

Reply:
Yeah, currently we don't use `NativeEventEmitter` anywhere in the codebase, so I think now it's better to keep `DeviceEventEmitter` and later (when needed) we can re-work all code pieces at once.